package.json

```diff
@@ -194,6 +194,7 @@
           "quotes": 0,
           "unicorn/catch-error-name": 0,
           "unicorn/no-console-spaces": 0,
+          "unicorn/regex-shorthand": 0,
```

Comment:
Added to avoid linting error on line 141 of webview.ts.

Reply:
@vsvipul we actually want this rule for improved readability of regex. The rule is also fixable so we can automate this.

Reply:
@priyank-p Great. I'll work on fixing this in webview then.

Reply:
Fixed this in the last commit. Please check if if is correct. Worked on regex for the first time.